Pipfile

```diff
@@ -26,7 +26,7 @@ uvicorn = "*"
 
 [packages]
 starlette = "==0.12.9"
-pydantic = "==1.0.0"
+pydantic = "==1.2"
```

Comment:
Is this change necessary? If not, probably better to remove it if only for the sake of clarity of what's changing. (It could go in a separate PR.)

Reply:
Understood.